MAINTAINERS.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+The oneDPL project provides algorithms that are implemented using parallel patterns and bricks implemented
+for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA platforms. The API areas
+which a developer is a lead maintainer are listed below. If you have questions, PRs, etc. which do not fit
+into one these categories you can check the git blame information to identify a contact or tag all maintainers.
+
+
+| Name                  | Github ID           | API Area |
+| --------------------- | ------------------- | -------- |
+| Ruslan Arutyunyan     | @rarutyun           | PSTL Offload |
+| Konstantin Boyarinov  | @kboyarinov         | PSTL Offload |
+| Pavel Dyakov          | @paveldyakov        | Random number generator APIs |
+| Mikhail Dvorskiy      | @MikeDvorskiy       | Range-based algorithms, Sort algorithms |
+| Adam Fidel            | @adamfidel          | Scan algorithms, Experimental Scan KT |
+| Dan Hoeflinger        | @danhoeflinger      | Histogram, Transform, Iterators |
+| Alexandr Konovalov    | @Alexandr-konovalov | PSTL Offload |
+| Sergey Kopienko       | @SergeyKopienko     | Find algorithms, Experimental Sort KT, validation tests |
+| Alexey Kukanov        | @akukanov           | oneDPL architecture, Experimental Sort KT |
+| Matt Michel           | @mmichel11          | By-segment algorithms, Search algorithms |
+| Julian Miller         | @julianmi           | Reduce algorithms |
+| Dmitriy Sobolev       | @dmitriy-sobolev    | Experimental Sort KT |
+| Anuya Welling         | @AnuyaWelling2801   | Experimental Dynamic Device Selection APIs |
+
```

Comment:
There are other areas, which we would also like to mention:

- Asynchronous algorithms: https://oneapi-src.github.io/oneDPL/parallel_api/async_api.html
- Probably, execution policies: https://oneapi-src.github.io/oneDPL/parallel_api/execution_policies.html
- Tested Standard API: https://oneapi-src.github.io/oneDPL/api_for_sycl_kernels_main.html
- Documentation
- CMake: https://oneapi-src.github.io/oneDPL/cmake_support.html
- Other integration methods (pkg-config, modulefiles, source scripts)
- Examples: https://github.com/oneapi-src/oneDPL/tree/main/examples

Should we have "other/default" category which would cover many other small areas, for example public macros?

Reply:
I think this has been covered in the offline discussion.  I don't know that I would name lead developers for documentation or examples.  In the latest version we don't have the tests covered, so I would prefer to leave the build system out.